generated/nidigital/session.py

```diff
@@ -807,6 +807,41 @@ def configure_voltage_levels(self, vil, vih, vol, voh, vterm):
         errors.handle_error(self, error_code, ignore_warnings=False, is_error_handling=False)
         return
 
+    @ivi_synchronized
+    def get_pin_results_pin_information(self):
+        '''get_pin_results_pin_information
+
+        Returns a list of named tuples (PinInfo) that <FILL IN THE BLANK HERE>
+
+        Fields in PinInfo:
+
+        - **pin_indexes** (int)
+        - **site_numbers** (int)
+        - **channel_indexes** (int)
+
+        Tip:
+        This method requires repeated capabilities (channels). If called directly on the
+        nidigital.Session object, then the method will use all repeated capabilities in the session.
+        You can specify a subset of repeated capabilities using the Python index notation on an
+        nidigital.Session repeated capabilities container, and calling this method on the result.:
+
+            session.channels[0,1].get_pin_results_pin_information()
+
+        Returns:
+            pin_info (list of PinInfo): List of named tuples with fields:
+
+                - **pin_indexes** (int)
+                - **site_numbers** (int)
+                - **channel_indexes** (int)
+
+        '''
+        import collections
+        PinInfo = collections.namedtuple('PinInformation', ['pin_indexes', 'site_numbers', 'channel_indexes'])
```

Comment:
Sorry for coming back with more comments. I'm discovering them as I'm writing system tests..

The fields names of namedtuple should be singular, e.g. `pin_index`

Reply:
Made singular